app/controllers/kms/public/pages_controller.rb

```diff
@@ -1,16 +1,16 @@
 module Kms
   class Public::PagesController < ActionController::Base
 
-    before_filter :eval_externals
+    before_action :eval_externals
 
     def show
-        page_result = Kms.template_manager.render(@page.register_id, @externals)
-        # apply typograph to page content
-        #page_result = EvilFront::Russian.typograph_html(page_result)
-        result   = Kms.template_manager.render(@template.register_id, @externals.
-                                merge(_inner_template: page_result))
+      page_result = Kms.template_manager.render(@page.register_id, @externals)
+      # apply typograph to page content
+      #page_result = EvilFront::Russian.typograph_html(page_result)
+      result   = Kms.template_manager.render(@template.register_id, @externals.
+                              merge(_inner_template: page_result))
 
-      render text: result, layout: false
+      render plain: result, content_type: "text/html"
```

Comment:
well any other options ends like that 
<img width="593" alt="screen shot 2017-01-22 at 12 13 22" src="https://cloud.githubusercontent.com/assets/342814/22182428/245a78b6-e09d-11e6-9e77-12be7cea1f51.png">
or like that
<img width="764" alt="screen shot 2017-01-22 at 12 14 53" src="https://cloud.githubusercontent.com/assets/342814/22182430/2c11efd0-e09d-11e6-80a6-0f1a3e4e7ffa.png">